Domain Connect Spec Draft.adoc

```diff
@@ -149,7 +149,10 @@ The template is defined by the Service Provider and manually onboarded with the
 Provider.
 Future versions of this specification may allow for an independent repository
 of templates. For now the templates are all published at
-https://github.com/Domain-Connect/templates
+https://github.com/Domain-Connect/templates. The filenames in this repository
+must be all lower case, including the providerId and serviceId. As a result,
+while the providerId and serviceId can be mixed case, all providerIds and
+serviceIds in this repository must be unique when lower case.

```

Comment:
I would prefer:

The filenames in this repository must be all lower case, including the providerId and serviceId. The values of the providerId and serviceId in the template and when passed into the API are case sensitive.